src/lib/spinel/spinel.h

```diff
@@ -1917,11 +1917,11 @@ enum
     SPINEL_PROP_CHANNEL_MONITOR_CHANNEL_OCCUPANCY = SPINEL_PROP_PHY_EXT__BEGIN + 10,
 
     /// Radio caps
-    /** Format: `i` (read-only)
+    /** Format: `S` (read-only)
      *
      * Data per item is:
      *
-     *  `i`: Radio Capabilities.
```

Comment:
The previous encoding say `i` which is "packed int" and should already be capable of handling `uint16` values.

So we should not need any changes?

Reply:
Right, somehow I read it as an 8 bit value. Just reverted it.